-Wdeprecated-missing-comma-variadic-parameter

The brand new clang-20 issues this warning under std=c++2c mode. Since it's a depreciation we'll need to comb through and fix all of these.

```
./boost/math/tools/traits.hpp:69:19: error: declaration of a variadic function without a comma before '...' is deprecated [-Werror,-Wdeprecated-missing-comma-variadic-parameter]
   69 | char cdf(const D& ...);
      |                   ^
      |                   , 
./boost/math/tools/traits.hpp:71:24: error: declaration of a variadic function without a comma before '...' is deprecated [-Werror,-Wdeprecated-missing-comma-variadic-parameter]
   71 | char quantile(const D& ...);
      |                        ^
      |                        , 
2 errors generated.
```

--- a/include/boost/math/tools/traits.hpp
+++ b/include/boost/math/tools/traits.hpp
@@ -66,9 +66,9 @@ constexpr bool has_backend_type_v = has_backend_type<T>::value;
 #endif
 
 template <typename D>
-char cdf(const D& ...);
+char cdf(const D&, ...);
 template <typename D>
-char quantile(const D& ...);
+char quantile(const D&, ...);
 
 template <typename D>
 struct has_cdf

